nixos/doc/manual/release-notes/rl-1709.xml

```diff
@@ -154,6 +154,15 @@ rmdir /var/lib/ipfs/.ipfs
       variables as parameters.
     </para>
   </listitem>
+  <listitem>
+    <para>
+      <literal>services.firefox.syncserver</literal> now runs by default as a
+      non-root user. You may have to fix permissions on the database and secret
+      file, or set it to run as root. To accomodate this change, the default
+      sqlite database location has also been changed. Refer to the description
+      of the options for more details.
```

Comment:
Since only the prestart script has root access, we need to set up the permissions before the service first starts. If no database exists, there is no file to chown so I put the db in a new folder so that we can chown the folder itself.